### ISSUE TITLE
Add pt, pt-BR and es-419 language support

### ISSUE THREAD

**Issue by ofrias** [gonativeio/gonative-ios#20]
I have added "pt-BR" and "es-419" to languages array in appConfig.json but their corresponding folders and not generated inside "LeanIOS" folder.  Could you please support these languages:

- pt
- pt-BR
- es-419

We need them to be included in the app to be able to configure the app description and screenshots for each language in the App Store.

**Comment by ah-abdullah** [comment]
Yes, we can add support for those languages. Please provide the localization strings for "Cancel", "OK", "View", "Menu"
e.g. https://github.com/gonativeio/gonative-ios/blob/6ef40dbd561744300c696ced071593403dac56ec/LeanIOS/gd.lproj/Localizable.strings

**Comment by ofrias** [comment]
es and es-419  (with regard to the current translation, please note that "Aceptar" is better than "OK" and the right word is "Menú" and not "Menu"):

/* iOS Button: Cancel */
"button-cancel" = "Cancelar";

/* iOS Button: OK */
"button-ok" = "Aceptar";

/* iOS Button: View */
"button-view" = "Ver";

/* iOS Button: Menu */
"button-menu" = "Menú";


pt and pt-BR:

/* iOS Button: Cancel */
"button-cancel" = "Cancelar";

/* iOS Button: OK */
"button-ok" = "Aceitar";

/* iOS Button: View */
"button-view" = "Assistir";

/* iOS Button: Menu */
"button-menu" = "Menu";
